changelog.yaml

```diff
@@ -30,6 +30,28 @@
       Feature flags can be configured via Helm Chart.
     pullRequestNumber: 178
 
+  - type: enhancement
+    impact: minor
+    title: Introduce network profiles
+    description: >-
+      Allow Steward clients to select a network profile per pipeline run.
+
+      Network profiles are named network-related configurations defined by Steward operators.
+      Currently they define a Kubernetes network policy. Additional network-related settings may be
+      added in the future.
+
+      Clients can choose a network profile via `spec.profiles.network` of a PipelineRun
+      resource object.
+    warning: ""
+    upgradeNotes: |-
+      - New Result `ResultErrorConfig` introduced.
+      - New feature flag `RetryOnInvalidPipelineRunsConfig` introduced.
```

Comment:
```suggestion
      - PiplineRun objects may now have a `status.result` value `error_config` (see Steward API docs).
      - Activating the new feature flag `RetryOnInvalidPipelineRunsConfig` will let the pipeline run controller retry reconciling in case the controller configuration (in ConfigMaps) is invalid or cannot be loaded. By default all PipelineRun objects will be set to finished with result code `error_infra`. The new behaviour will become the default in a future release of Steward.
```